src/resource/factory/SoH/PlayerAnimationFactory.cpp

```diff
@@ -8,8 +8,8 @@ PlayerAnimation* PlayerAnimationFactory::ReadPlayerAnimation(BinaryReader* reade
 
     switch (version) {
         case Version::Deckard: {
-            PlayerAnimationV0 Anim = PlayerAnimationV0();
-            Anim.ParseFileBinary(reader, anim);
+            PlayerAnimationV0 animFac = PlayerAnimationV0();
```

Comment:
probably sensing a theme here, i don't like the `Fac` naming convention

Reply:
Out of scope.